controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -1161,6 +1188,243 @@ func saveHostProvisioningSettings(host *metal3v1alpha1.BareMetalHost) (dirty boo
 	return
 }
 
+// Get the stored firmware settings and validate the Spec settings against the schema
+func (r *BareMetalHostReconciler) getHostFirmwareSettings(prov provisioner.Provisioner, info *reconcileInfo) (hfs *metal3v1alpha1.HostFirmwareSettings, err error) {
+
+	hfs = &metal3v1alpha1.HostFirmwareSettings{}
+	if r.Get(context.TODO(), info.request.NamespacedName, hfs) != nil {
+		// Could not get settings, don't return error as settings may not have been available at provisioner
```

Comment:
Maybe limit the number of ignored errors? E.g. ignore "not found", but not connection failures?

Reply:
Fixed to return an error if the Get fails but it isn't a NotFound returned.